### PR TITLE
Update set-password-expiration-policy.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to uncheck the box instead of select the check box

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3559

File: microsoft-365/admin/manage/set-password-expiration-policy.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Follow the steps below if you want to set user passwords to expire after a speci
   
 3. Select **Password expiration policy**.
   
-4. If you don't want users to have to change passwords, select the checkbox next to **Set user passwords to expire after a number of days**.
+4. If you don't want users to have to change passwords, Uncheck the box next to **Set user passwords to expire after a number of days**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/set-password-expiration-policy.md]:
```suggestion
4. If you don't want users to have to change passwords, uncheck the box next to **Set user passwords to expire after a number of days**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**